INSTALL.md

```diff
@@ -1,16 +1,35 @@
 # Install
 
+## Quick Start
+
+If you just want to try out the astrobee simulator, you can use one of the docker images in the [Docker Hub](https://hub.docker.com/r/astrobee/astrobee). Make sure you have docker installed in your ubuntu system flolowing the [installation instructions](https://docs.docker.com/engine/install/ubuntu/) and [post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/).
+Given that Docker is installed, just run:
+
+    ./scripts/docker/run.sh
+
+Be aware that this only works on a native ubuntu install.
+
+## Building the code natively
+
 There are different ways to build and test the code.
 
+### Non-NASA users
```

Comment:
Right now these build instructions have three equal branches "non-NASA", "NASA", "Docker". But everyone is either non-NASA or NASA, so who will keep reading down to "Docker"? I think the tree structure should be "Docker" vs. "Native" and then within "Native" comes the "non-NASA" and "NASA". It would help to give some hint about the pros and cons of "Docker" vs. "Native".

Reply:
Changed the level on the branches and made "Native" and "Docker" the same level.